flink-docs/src/main/java/org/apache/flink/docs/rest/RestAPIDocGenerator.java

```diff
@@ -220,32 +220,42 @@ private static String createHtmlEntry(MessageHeaders<?, ?, ?> spec) {
         {
             sb.append("    <tr>\n");
             sb.append("      <td colspan=\"2\">\n");
-            sb.append(
-                    "        <button data-toggle=\"collapse\" data-target=\"#"
-                            + reqHash
-                            + "\">Request</button>\n");
-            sb.append("        <div id=\"" + reqHash + "\" class=\"collapse\">\n");
+            sb.append("      <div class=\"book-expand\">\n");
+            sb.append("        <label>\n");
+            sb.append("          <div class=\"book-expand-head flex justify-between\">\n");
+            sb.append("            <span>Request</span>\n");
+            sb.append("            <span>...</span>\n");
```

Comment:
How about:

`sb.append("            <span>▾</span>\n");`

Reply:
actually, why not just a single `span` without the `justify-between`? Having the dots/arrow not jump to the right when opened seems nicer?